src/main/scala/org/broadinstitute/dsde/workbench/avram/api/AvramRoutes.scala

```diff
@@ -13,4 +25,35 @@ class AvramRoutes {
   def ping: Pong = {
     Pong()
   }
+
+  // TODO: remove this endpoint when we have more meaningful ways to test database queries
+  @ApiMethod(name = "now", httpMethod = "get", path = "now")
+  def now: Now = {
+    // Explicitly use a Future to make sure the implicit ExecutionContext is being used
+    Now(Await.result(Future(fetchTimestampFromDBWithSlick()), Duration.Inf))
+  }
+
+  // TODO: move/merge this endpoint into a status API
+  @ApiMethod(name = "dbPoolStats", httpMethod = "get", path = "dbPoolStats")
+  def dbPoolStats: DbPoolStats = {
+    val result = for {
+      totalConnections <- DataSource.database.run(
+        sql"select count(*) from pg_stat_activity where pid <> pg_backend_pid() and usename = current_user".as[Int])
+    } yield DbPoolStats(DataSource.ds.getNumActive, DataSource.ds.getNumIdle, totalConnections.head)
+    Await.result(result, Duration.Inf)
+  }
+
+  private def fetchTimestampFromDBWithSlick(): String = {
+    val now = Await.result(DataSource.database.run(sql"select now()".as[String]), Duration.Inf)
+    now.head
+  }
+
+  private def fetchTimestampFromDBWithJDBC(): String = {
+    val config = ConfigFactory.load().getConfig("postgres")
+    val conn: Connection = DriverManager.getConnection(config.getString("url"), config.getString("user"), config.getString("password"))
```

Comment:
why doesn't this use the datasource?

Reply:
I should delete this. It's left over from when I thought I was seeing some slick-specific problems.